.github/workflows/release-doc.yml

```diff
@@ -0,0 +1,24 @@
+name: Publish release changelog
+on:
+  push:
+    tags:
+      - v[0-9]+.[0-9]+.[0-9]+*
+jobs:
+  release:
+    if: startsWith(github.ref, 'refs/tags/')
+    runs-on: ubuntu-latest
+    steps:
+      - name: Build Changelog
+        id: pmm_release
+        shell: bash
+        run: |
+          git clone --depth 1 https://github.com/percona/pmm-doc doc
+          cd doc/docs/release-notes
+          version="${release_tag/refs\/tags\/v//}"
+          cat ./$version.md > ${{ github.workspace }}-CHANGELOG.txt
```

Comment:
It looks like we can upload `$version.md` directly.

Reply:
Nikita suggested using raw. I simplified it and do not use git clone.

Reply:
I mean what is the reason to rename file to `*-CHANGELOG.txt` ? It's still Markdown.

Reply:
It's still markdown, but ${{ github.workspace }} is a directory. To make it predictable between steps I add `-CHANGELOG.txt` and upload it using the next pipeline step.